src/ElectricType.java

```diff
@@ -0,0 +1,26 @@
+public class ElectricType extends Pokemon {
+    private String electricAttack1;
+    private String electricAttack2;
+    public ElectricType(String name, String type, int level, int hp, int xp, String sound, String attack, String electricAttack1, String electricAttack2){
```

Comment:
Enige opmerking die ik heb: probeer iets netter om te gaan met je witregels. Zorg dat er een wit regel zit tussen je properties en je constrcutor. En hou ook de constantheid dat het steeds maar 1 wit regel is en niet soms meerdere.

Qua code klopt het allemaal!